pkg/scheduler/scheduler.go

```diff
@@ -109,6 +110,13 @@ func WithProfiles(p ...schedulerapi.KubeSchedulerProfile) Option {
 	}
 }
 
+// WithParallelism sets the parallelism for all scheduler algorithms. Default is 16.
+func WithParallelism(threads int32) Option {
+	return func(o *schedulerOptions) {
+		parallelize.SetParallelism(int(threads))
```

Comment:
add a `TODO(name or issue)` here to move to schedulerOptions and stop calling the global setter here

Reply:
```
TODO(#95952): Remove global setter in favor of a struct that holds the configuration.
```

Reply:
Still not solved